test/e2e/framework/framework.go

```diff
@@ -182,8 +180,6 @@ func (f *Framework) BeforeEach() {
 		}
 		f.ClientSet, err = clientset.NewForConfig(config)
 		ExpectNoError(err)
-		f.AggregatorClient, err = aggregatorclient.NewForConfig(config)
-		ExpectNoError(err)
 		f.DynamicClient, err = dynamic.NewForConfig(config)
```

Comment:
just one question for next work: I guess we would keep this DynamicClient because the callers are 17 like
```
test/e2e$ grep f.DynamicClient * -R | wc -l
17
```
right?

Reply:
i think we should keep it - this dynamic client is part of client-go.
we only had agreement with api-machinery to move the aggregator.

Reply:
@neolit123 Thanks for your explanation, that seems reasonable move :-)